framer/Print.coffee

```diff
@@ -35,7 +35,7 @@ class Printer
 
 			@_closeButton = new Layer
 				parent: @_container
-				html: '<svg><g stroke="#B8B8B8"><path d="M1,1 L8,8"></path><path d="M1,8 L8,1"></path></g></svg>'
+				html: '<svg viewBox="0 0 9 9"><g stroke="#B8B8B8"><path d="M1,1 L8,8"></path><path d="M1,8 L8,1"></path></g></svg>'
```

Comment:
Thanks. I reverted this change and also noticed I see the same sizing issue in Chrome (using the Project template in the repo), the small close button sizes to 300x150!

![svgwbug](https://cloud.githubusercontent.com/assets/1250894/23175367/78ab74a4-f85f-11e6-815c-0146ed1c5811.png)

Couldn't figure out why, but setting **overflow: hidden** on the body tag seems like a workaround.